core/src/test/scala/unit/kafka/server/ISRExpirationTest.scala

```diff
@@ -66,7 +66,7 @@ class IsrExpirationTest {
    */
   @Test
   def testIsrExpirationForStuckFollowers() {
```

Comment:
This and the other test started failing because the following line was removed:

https://github.com/apache/kafka/pull/1985/files#diff-5450baca03f57b9f2030f93a480e6969L418

Got to love mocks. :)